README.md

```diff
@@ -63,12 +63,23 @@ By default, `nixci` will build the top-level flake, but you can tell it to build
 # myproject/flake.nix
 {
   nixci = {
-    dir1 = {
-        dir = "dir1";
+    # This is the default configuration which will be used
+    default = {
+        dir1 = {
+            dir = "dir1";
+        };
+        dir2 = {
+            dir = "dir2";
+            overrideInputs.myproject = ./.;
+        };
     };
-    dir2 = {
-        dir = "dir2";
-        overrideInputs.myproject = ./.;
+
+    # Alternative configurations can be defined and subsequently selected via
+    # `nixci --config extra-tests .`
```

Comment:
Do we really need a new option (`--config`)? Why not `nixci .#extra-tests`? 

As well, `nixci https://github.com/srid/nixci/pull/20#extra-tests`?

Reply:
That's a good suggestion, I hadn't really considered it!

On first reaction I would worry whether having `nixci .#extra-tests` would be confusing (i.e. that it's really `nixci.extra-tests` in the flake) but there are other tools which do this so there's already a precedent somewhat established. Let me try taking the explicit option out

Reply:
Done!